packages/bits/src/lib/table/table-cell/table-cell.directive.ts

```diff
@@ -3,6 +3,7 @@ import { ChangeDetectorRef, Directive, ElementRef, HostBinding, HostListener, In
 import _get from "lodash/get";
 import { Subscription } from "rxjs";
 import { filter } from "rxjs/operators";
+import {FIXED_WIDTH_CLASS} from "../constants";
```

Comment:
It doesn't look like this file is using this constant. Should we be using this constant as a filter below?